package.json

```diff
@@ -50,6 +50,7 @@
     "helmet": "^6.0.1",
     "is-ip": "^5.0.0",
     "isbot": "^3.6.5",
+    "random-words": "^1.2.1",
```

Comment:
I don't think we need a dependency for this.  Can you use something like:

```ts
const randomWord = (function () {
  let seed = Date.now();
  return () => String(seed++);
}());
```

Reply:
oh, I didn't think about it. I will remove the dependency.

Reply:
This should get removed, it looks like you still have it?